dev/fattest.simplicity/build.gradle

```diff
@@ -63,10 +65,30 @@ task extractArquillianSupportFeature(type:Sync) {
     }
 }
 
+task extractArquillianSupportJakartaFeature(type:Sync) {
+    configurations.arquillianJakartaFeature.each {
+        from(zipTree(it))
+    }
+    rename { String mfFile ->
+        mfFile.replace("arquillian-liberty-support.mf", "arquillian-liberty-support-jakarta.mf")
+    }
+    into('build/arquillian-jakarta-feature-extract')
+
+    doLast {
+        // Update the feature manifest to mark it as a test feature
+        file('build/arquillian-jakarta-feature-extract/extension/lib/features/arquillian-liberty-support-jakarta.mf')
+            .append('IBM-Test-Feature: true\n')
+    }
+}
+
 task publishArquillianSupportFeature(type:Copy) {
     dependsOn extractArquillianSupportFeature
     from('build/arquillian-feature-extract/extension')
     into(buildImage.file('wlp'))
+
+    dependsOn extractArquillianSupportJakartaFeature
+    from('build/arquillian-jakarta-feature-extract/extension')
+    into(buildImage.file('wlp'))
```

Comment:
I don't think you need two `into`s here. I think the one above will apply to the whole task.